src/views/hooks/useCronTabActionsProvider.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { CronTabKind } from 'src/types';
+import { Action } from '@openshift-console/dynamic-plugin-sdk';
+
+type UseCronTabActionsProvider = (
+  cronTab: CronTabKind,
+) => [actions: Action[]];
+
+export const useCronTabActionsProvider: UseCronTabActionsProvider = (cronTab) => {
+
+  const actions = React.useMemo(
+    () => [],
+    [],
```

Comment:
If there are no actions, I'd leave this out. We can keep this in a separate branch so we still have the code.

Reply:
removed